suite-native/module-accounts-import/src/components/XpubHintSheet.tsx

```diff
@@ -0,0 +1,90 @@
+import React, { ReactNode, useState } from 'react';
+import { TouchableOpacity, View } from 'react-native';
+
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+import { BottomSheet, Box, Button, Text, VStack } from '@suite-native/atoms';
+import { Icon } from '@trezor/icons';
+import { NetworkSymbol } from '@suite-common/wallet-config';
+import { Video } from '@suite-native/video-assets';
+
+type XpubScanHintSheet = {
+    currencySymbol: NetworkSymbol;
+};
+
+const sheetTriggerStyle = prepareNativeStyle(_ => ({
+    flexDirection: 'row',
+    width: '100%',
+    justifyContent: 'center',
+    alignItems: 'center',
+}));
+
+const coinToTitleMap = {
+    btc: 'Where is my public key (XPUB)?',
+    eth: 'Where is my receive address?',
+    xrp: 'Where is my receive address?',
+} as const as Record<NetworkSymbol, string>;
+
+const EmphasizedText = ({ children }: { children: ReactNode }) => (
+    <Text color="gray900">{children}</Text>
+);
+
+const coinToTextMap = {
+    btc: (
+        <>
+            To view the Bitcoin XPUB of your account, open the Trezor Suite desktop app, plug in
+            your Trezor device, select <EmphasizedText>Accounts</EmphasizedText>, then choose{' '}
+            <EmphasizedText>Show public key</EmphasizedText>.
+        </>
+    ),
+    eth: (
+        <>
+            To view the Ethereum and Ethereum Classic receive address of your account, open the
+            Trezor Suite desktop app, plug in your Trezor device, select{' '}
+            <EmphasizedText>Accounts</EmphasizedText>, choose Receive, and click on{' '}
+            <EmphasizedText>Show full address</EmphasizedText>.
+        </>
+    ),
+    xrp: (
+        <>
+            To view the Ripple receive address of your account, open the Trezor Suite desktop app,
+            plug in your Trezor device, select <EmphasizedText>Accounts</EmphasizedText>, choose
+            <EmphasizedText>Receive</EmphasizedText>, and click on{' '}
+            <EmphasizedText>Show full address</EmphasizedText>
+        </>
+    ),
+} as const as Record<NetworkSymbol, ReactNode>;
+
+export const XpubHintSheet = ({ currencySymbol }: XpubScanHintSheet) => {
```

Comment:
`XpubHintModal`

Reply:
fixed aa34c4297

Reply:
@Nodonisko we are using sheet naming throughout the whole app. Why change it here?